lib/ueberauth/strategy/facebook.ex

```diff
@@ -56,6 +53,22 @@ defmodule Ueberauth.Strategy.Facebook do
       fetch_user(conn, client)
     end
   end
+  def handle_callback!(token) when is_binary(token) do
```

Comment:
This is more about depending of `Plug.Conn` or not. Which I would push for avoid it if I can, so I can send that token throw `web socket` terminal. It doesn't matter, the module is flexible